packages/graphql/tests/tck/connections/alias.test.ts

```diff
@@ -141,4 +141,136 @@ describe("Connections Alias", () => {
             }"
         `);
     });
+
+    test("should alias startCursor", async () => {
```

Comment:
Should these be tck tests?

All of these aliases are done in JS, which means that the cypher itself is unchanged. 

These seem to be present in the file `alias.int.test.ts`, any reason for adding them here?